cfgen/cfgen

```diff
@@ -892,17 +920,18 @@ class LibfabricEndpoint(Endpoint):
     ep_map: Dict[Tuple[str, int], int] = {}
 
     def __init__(self, NetFamily: str, proto: NetProtocol, ipaddr: str,
-                 portalgroup: LibfabPortalGroup):
+                 portalgroup: LibfabPortalGroup, proct: str):
         self.netfamily = NetFamily
         self.proto = proto
         assert ipaddr
         self.ipaddr = ipaddr
-        assert portalgroup.value > 0
+        assert portalgroup
         self.portalgroup = portalgroup
         self.portal = 0
-        self.portal = self.ep_map.setdefault((ipaddr, portalgroup.value),
-                                             (portalgroup.value + 1))
-        self.ep_map[(ipaddr, portalgroup.value)] += 1
+        port_range = portalgroup.__dict__[proct]
```

Comment:
This will change as per above mentioned implementation, no need of additional argument `proct`.